astropy/visualization/wcsaxes/patches.py

```diff
@@ -81,9 +83,15 @@ class SphericalCircle(Polygon):
 
     def __init__(self, center, radius, resolution=100, vertex_unit=u.degree, **kwargs):
 
-        # Extract longitude/latitude, either from a tuple of two quantities, or
-        # a single 2-element Quantity.
-        longitude, latitude = center
+        # Extract longitude/latitude, either from a SkyCoord object, or
+        # from a tuple of two quantities or a single 2-element Quantity.
+        if isinstance(center, SkyCoord):
+            if center.frame.name == 'galactic':
+                longitude, latitude = center.l, center.b
```

Comment:
OK, now, really last thing: I started the tests again and this line is not covered. Can you add another test? I don't think you need to draw anything for the test, so that we don't have to add an image. Just make any example locally so that you see it works and then you just test a single point similar to "assert np.allclose((r1.get_xy()[0], [123, 456])" where you obviously need to replace the 123 and 456 with whatever  number you get in the example.

You'll see that I approved it already, so it could be merged right now, but that little addition (just to project us form something stupid, say, accidentially switching l and b) would make it even better. Thank you so much for your patiance!

Reply:
I'll try to add that :)